store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9629,4 +9629,9 @@ TODO: delete them permanently from here
     <desc>Account thumbnail photo</desc>
 </attr>
 
+<attr id="3022" name="zimbraLDAPSchemaVersion" type="string" cardinality="single" requiredIn="globalConfig" since="8.8.8">
```

Comment:
Note that you are going to need to choose a new id for this.  I think this one is taken now...